lbrynet/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 log = logging.getLogger(__name__)
 logging.getLogger(__name__).addHandler(logging.NullHandler())
-log.setLevel(logging.ERROR)
+log.setLevel(logging.INFO)
```

Comment:
This is a note to me to delete this section - as with my other logging pull, log configuration is better handled by the caller, not at import time.